Blish HUD/GameServices/Overlay/DynamicHUDMethod.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel;
+
+namespace Blish_HUD.Graphics {
+    public enum DynamicHUDMethod : int {
+        [Description("Always Show (Default)")]
+        AlwaysShow = 0,
+
+        [Description("Show Only out of Combat")]
+        ShowPeaceful = 1,
+
+        [Description("Show Only in Combat")]
+        ShowInCombat = 2,
+
+        [Description("Never Show")]
+        NeverShow = 3
```

Comment:
Same goes with `NeverShow` for the corner icons.  It's a quick way to get soft locked without an easy way to fix the setting.